service/src/main/java/com/astasko/pdp/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.astasko.pdp.service.impl;
+
+import com.astasko.pdp.domain.model.User;
+import com.astasko.pdp.repository.api.UserRepository;
+import com.astasko.pdp.service.api.UserService;
+
+import javax.annotation.Resource;
+
```

Comment:
Add @Service annotation instead of context configuration. SpringBoot will configure automatically